app/modbus.mjs

```diff
@@ -65,6 +65,22 @@ export const setFlag = async (modbusClient, flag, value) => {
     }
 
     await mutex.runExclusive(async () => modbusClient.writeCoil(AVAILABLE_FLAGS[flag], value))
+
+    // Flags are mutually exclusive, disable all others when enabling one
+    if (value) {
+        await disableAllModesExcept(modbusClient, flag)
+    }
+}
+
+const disableAllModesExcept = async (modbusClient, exceptedMode) => {
+    for (const mode in AVAILABLE_FLAGS) {
+        // summerNightCooling can be enabled simultaneously with other modes
+        if (mode === exceptedMode || mode === 'summerNightCooling') {
```

Comment:
Would prefer predefined const

Reply:
Yeah, I'll refactor this a bit. "flag" is a legacy misnomer, the upper layers talk about "modes" instead. I think summer night cooling has to be moved away from the list of available modes since it doesn't fit into the model.

Reply:
Sounds good.

Reply:
Made a separate list for mutually exclusive modes